src/js/dates.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * DATES
+ *
+ * @package uri-component-library
+ */
+
+( function() {
+	'use strict';
+
+	window.addEventListener( 'DOMContentLoaded', init, false );
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['init' was used before it was defined.](https://app.codacy.com/manual/uriweb/uri-component-library/pullRequest?prid=4835245)